### ISSUE TITLE
No Ext.Ajax.request

### ISSUE THREAD

**Issue by bedorlan** [thanhptr/extts#4]
No Ext.Ajax.request

`
    export class Ajax {
    }
`

**Comment by maxtpham** [comment]
Yes, may be, cos i'm using JQuery.Ajax so i don't know its existence.

The generation has not completed yet, and while using the def, we have to fix some.

I have just publish the source, so welcome everyone to look into this and make a pull request to make the def better for the community.
